src/js/models/StopDetails.js

```diff
@@ -40,12 +34,8 @@ StopDetails.prototype.fetch = function() {
     function retryAtMost(maxRetries) {
         requests.get(yqlURL, params)
             .then(this.parseResponse.bind(this))
-            .tap(function(Services) {
-                var tripCollections = Services.map(function(Service) {
-                    return new TripCollection(this.stopID(), Service);
-                }.bind(this));
-
-                this.tripCollections(tripCollections);
+            .tap(function(Runs) {
+                this.tripCollection = new TripCollection(this.stopID(), this.routeID(), Runs[0].Sign, Runs);
```

Comment:
Why don't we just remove the `sign` param from the TripCollection constructor and let the constructor do the `this.sign = Runs[0]`.